runtime/astar/src/xcm_config.rs

```diff
@@ -279,6 +279,7 @@ impl xcm_executor::Config for XcmConfig {
     type HrmpNewChannelOpenRequestHandler = ();
     type HrmpChannelAcceptedHandler = ();
     type HrmpChannelClosingHandler = ();
+    type XcmRecorder = ();
```

Comment:
Shouldn't this be set to `PolkadotXcm`? 🤔 

You rely on the `PolkadotXcm` implementation of dry run functions, but internally they rely on this particular type - in this case nothing will be recorded since the empty default implementation is used.

Reply:
Nice catch you're right, thanks! I've totally forgot to adjust production runtimes, now is done ✅